sdk/maps/azure-maps-search/src/test/java/com/azure/maps/search/TestUtils.java

```diff
@@ -0,0 +1,307 @@
+package com.azure.maps.search;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.models.GeoLineString;
+import com.azure.core.models.GeoObject;
+import com.azure.core.test.TestBase;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.SerializerAdapter;
+import com.azure.core.util.serializer.SerializerEncoding;
+import com.azure.core.util.serializer.TypeReference;
+import com.azure.maps.search.implementation.helpers.Utility;
+import com.azure.maps.search.implementation.models.PolygonPrivate;
+import com.azure.maps.search.implementation.models.ReverseSearchAddressResultPrivate;
+import com.azure.maps.search.implementation.models.ReverseSearchCrossStreetAddressResultPrivate;
+import com.azure.maps.search.implementation.models.SearchAddressResultPrivate;
+import com.azure.maps.search.models.BatchReverseSearchResult;
+import com.azure.maps.search.models.BatchSearchResult;
+import com.azure.maps.search.models.PointOfInterestCategoryTreeResult;
+import com.azure.maps.search.models.Polygon;
+import com.azure.maps.search.models.ReverseSearchAddressResult;
+import com.azure.maps.search.models.ReverseSearchCrossStreetAddressResult;
+import com.azure.maps.search.models.SearchAddressResult;
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.provider.Arguments;
+
+
+public class TestUtils {
+
+    static final String FAKE_API_KEY = "1234567890";
+    public static final Duration DEFAULT_POLL_INTERVAL = Duration.ofSeconds(30);
+
+    static Polygon getPolygon(File file) {
+        SerializerAdapter jacksonAdapter = JacksonAdapter.createDefaultSerializerAdapter();
+        TypeReference<PolygonPrivate> interimType = new TypeReference<PolygonPrivate>(){};
+        byte[] data = null;
+        try {
```

Comment:
let exceptions bubble out of the method, don't try/catch anything here. calling code (the test) needs to deal with it